scripts/process_linnarsson.sh

```diff
@@ -95,33 +95,30 @@ process_linnarson_images() {
         [ -e "$HDF5_IN" ] || \
             wget "$PKLAB_URL/Nuclei_polyT.int16.sf.hdf5" -O "$HDF5_IN"
 
+        RELEASE=${S3_TARGET//vitessce-data\//}
+        ZARR_OUT='linnarsson.images.zarr'
+        ZARR_URL="https://vitessce-data.storage.googleapis.com/$RELEASE/linnarsson/$ZARR_OUT"
+
         CMD="$BASE/python/img_hdf5_reader.py
             --hdf5 $HDF5_IN
-            --channel_clip_pairs polyT:200 nuclei:20
             --sample 1
-            --json_file $JSON_OUT"
+            --channels polyT,nuclei
+            --json_file $JSON_OUT
```

Comment:
Can you change `$JSON_OUT` to be `linnarsson.raster.json` instead of `linnarsson.images.json`?

Reply:
Sorry I forgot about this, but I would rather not maintain two schemas.  I can change the file name on s3.

Reply:
I asked about this earlier ... sure

Reply:
Yes, sorry about that.

Reply:
it wont be nested in the zarr storage. is that alright?

Reply:
`vitessce-data/0.0.23/master_release/linnarsson/linnarsson.raster.json`